content/influxdb/v2.2/write-data/troubleshoot.md

```diff
@@ -60,25 +77,240 @@ Write requests return the following status codes:
-For example, did you attempt to write `string` data to an `int` field?
+{{% cloud-only %}}
+If you notice data is missing in your bucket, do the following:
+
+- Check the `message` property in the response body for details about the error--for example, `partial write error` indicates [rejected points](#troubleshoot-rejected-points).
+- Check for [rejected points](#troubleshoot-rejected-points) in your organization's `_monitoring` bucket.
+- Verify all lines contain valid syntax ([line protocol](/influxdb/cloud/reference/syntax/line-protocol/) or [CSV](/influxdb/cloud/reference/syntax/annotated-csv/)). See how to [find parsing errors](#find-parsing-errors).
+- Verify the data types match the [series](/influxdb/cloud/reference/key-concepts/data-elements/#series) or [bucket schema](/influxdb/cloud/organizations/buckets/bucket-schema/). See how to resolve [explicit schema rejections](#resolve-explicit-schema-rejections).
+- Verify the timestamps match the [precision parameter](/influxdb/cloud/write-data/#timestamp-precision).
+- Minimize payload size and network errors by [optimizing writes](/influxdb/cloud/write-data/best-practices/optimize-writes/).
+{{% /cloud-only %}}
+
+## Troubleshoot rejected points
+
+{{% oss-only %}}
 
 InfluxDB rejects points for the following reasons:
+
 - The **batch** contains another point with the same series, but one of the fields has a different value type.
 - The **bucket** contains another point with the same series, but one of the fields has a different value type.
+
+Check for [field type](/influxdb/v2.2/reference/key-concepts/data-elements/#field-value) differences between the missing data point and other points that have the same [series](/influxdb/v2.2/reference/key-concepts/data-elements/#series)--for example, did you attempt to write `string` data to an `int` field?
+
+{{% /oss-only %}}
+
+{{% cloud-only %}}
+
+InfluxDB may have rejected points even if the HTTP request returned "Success".
+InfluxDB logs rejected data points and associated errors to your organization's `_monitoring` bucket.
+
+- [Review rejected points](#review-rejected-points)
+  - [Find parsing errors](#find-parsing-errors)
+  - [Find data type conflicts and schema rejections](#find-data-type-conflicts-and-schema-rejections)
+- [Resolve data type conflicts](#resolve-data-type-conflicts)
+- [Resolve explicit schema rejections](#resolve-explicit-schema-rejections)
+
+### Review rejected points
+
+To get a log of rejected points, query the [`rejected_points` measurement](/influxdb/cloud/reference/internals/system-buckets/#_monitoring-bucket-schema) in your organization's `_monitoring` bucket.
+To more quickly locate `rejected_points`, keep the following in mind:
+
+- If your line protocol batch contains single lines with multiple [fields](/influxdb/cloud/reference/syntax/line-protocol/#field-set), InfluxDB logs an entry for each point (each unique field) that is rejected.
```

Comment:
Is the limiting factor _both_ single lines and multiple fields in a single line? 

For example, what happens if there's one long line with multiple fields?

Reply:
> Is the limiting factor _both_ single lines and multiple fields in a single line?
> 
> For example, what happens if there's one long line with multiple fields?

Is this better:

If your line protocol batch contains multiple [fields](/influxdb/cloud/reference/syntax/line-protocol/#field-set) per line, InfluxDB logs an entry for each point (each unique field) that is rejected.